WordPress/wordpress-playground
# [Remote] Remove the "light" PHP.wasm bundle and only ship the "kitchen-sink" build

Maintaining the "light" PHP.wasm bundle is a burden (#1848), especially with the JSPI support on the horizon (#1339). Only 1.3% of Playgrounds are loaded with the `light` extension bundle (see #1848).

Let's thus simplify the project maintenance and remove the "light" bundle. After this PR, every Playground will be loaded with the "kitchen-sink" extension bundle. This PR also drops the name "kitchen-sink". From now on, it's just "php",

 ## Developer notes

The web version of Playground at playground.wordpress.net no longer ships the `light` build of PHP.wasm. Instead, it only ships the `kitchen-sink` build where popular PHP extensions are available (e.g. libxml, libopenssl). Note that the `kitchen-sink` build was already used by 98.7% of Playgrounds and most users are not affected by this.

This change will help Playground maintainers focus on a single version and enable solving a lot of PHP.wasm stability issues by [migrating to JSPI](https://github.com/WordPress/wordpress-playground/pull/1339). 

The CLI version of Playground is unaffected by this change.

The following APIs are now deprecated. Playground will accept them without crashing, but they will no longer have any effect:

* `phpExtensionBundles` Blueprint setting
* `php-extension-bundle` Query API parameter

## Follow up work

- [ ] Publish the developer notes on https://make.wordpress.org/playground/
- [x] Share this in the WP.org Slack channel ([done](https://wordpress.slack.com/archives/C04EWKGDJ0K/p1728405828152549))

 ## Testing instructions

* Confirm Playground continues to work in the local dev env
* Confirm the CI is green – it would perform a thorough test run of many user flows.

Closes #1848
